data/json/items/generic.json

```diff
@@ -1503,6 +1505,7 @@
         "name": "science ID card",
         "description": "This ID card once belonged to a scientist of some sort.  It has a magnetic stripe on the back; perhaps it can be used on a control panel.",
         "price": 60000,
+        "price_postapoc": 5000,
```

Comment:
I'm not sure about this: ID card is your pass to treasure (or to graveyard, but let's be optimistic), so I see no reason why it worth less than in pre-apoc times.